exercises/practice/error-handling/.meta/proof.ci.js

```diff
@@ -0,0 +1,9 @@
+export const processString = (input) => {
+  if (typeof input !== 'string') {
+    throw new TypeError();
+  }
+  if (input === '') {
+    throw new Error();
```

Comment:
We almost always want a message in an error. We don't need to test the exact message, but throwing an empty error is non-idiomatic.